pkg/modules/logging/logging.go

```diff
@@ -198,6 +198,16 @@ func newLogEncoder(format string, gcpSeverity bool) (zapcore.Encoder, error) {
 
 		if gcpSeverity {
 			encCfg.EncodeLevel = gcpSeverityEncoder
+			encCfg.TimeKey = "time"
+			encCfg.LevelKey = "severity"
+			encCfg.NameKey = "logger"
+			encCfg.CallerKey = "caller"
+			encCfg.MessageKey = "message"
+			encCfg.StacktraceKey = "stacktrace"
+			encCfg.LineEnding = zapcore.DefaultLineEnding
+			encCfg.EncodeTime = zapcore.RFC3339TimeEncoder
+			encCfg.EncodeDuration = zapcore.MillisDurationEncoder
+			encCfg.EncodeCaller = zapcore.ShortCallerEncoder
```

Comment:
```suggestion
			// Those only make sense in JSON.
			encCfg.TimeKey = "time"
			encCfg.LevelKey = "severity"
			encCfg.MessageKey = "message"
			encCfg.EncodeTime = zapcore.ISO8601TimeEncoder
			encCfg.EncodeDuration = zapcore.MillisDurationEncoder
```

Other fields are not necessary, as those are already the default values of `zap.NewProductionEncoderConfig()` 👍

Reply:
Regarding `zapcore.RFC3339TimeEncoder` vs `zapcore.ISO8601TimeEncoder`: https://github.com/uber-go/zap/discussions/1110#discussioncomment-4461995